Dockerfile.deploy

```diff
@@ -41,7 +40,6 @@ RUN touch /addons-server-docker-container \
         zlib1g-dev \
         libffi-dev \
         libssl-dev \
-        libpcre3-dev \
```

Comment:
this is listed twice fwiw - are you trying to drop it or removing the duplicate?

Reply:
I intend to drop it because https://github.com/mozilla/addons-server/pull/8143 added it and this PR is to undo that PR.

Reply:
Now it's dropped.